.travis.yml

```diff
@@ -1,8 +1,14 @@
 dist: trusty
 sudo: false
+
```

Comment:
I think you can get rid of `dist: trusty` and change to `sudo: required` as per the [docs](https://docs.travis-ci.com/user/docker/)